src/main/java/kitchenpos/domain/MenuProducts.java

```diff
@@ -0,0 +1,39 @@
+package kitchenpos.domain;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@Embeddable
+public class MenuProducts {
+    @OneToMany(mappedBy = "menu", fetch = FetchType.LAZY, cascade = CascadeType.PERSIST)
+    private List<MenuProduct> menuProducts = new ArrayList<>();
+
+    protected MenuProducts() {}
+
+    public MenuProducts(List<MenuProduct> menuProducts) {
+        this.menuProducts = new ArrayList<>(menuProducts);
+    }
+
+    public List<MenuProduct> getList() {
+        return this.menuProducts;
+    }
+
+    public BigDecimal getSumPrice() {
+        BigDecimal sum = BigDecimal.ZERO;
+        for (MenuProduct menuProduct : menuProducts) {
+            sum = sum.add(menuProduct.getProduct()
+                    .getPrice().multiply(BigDecimal.valueOf(menuProduct.getQuantity())));
```

Comment:
```suggestion
            sum = sum.add(menuProduct.price());
```
`menuProduct`의 가격은 `menuProduct`에서 계산하는게 좋지 않을까요? 🤔